src/app/Events/Events.tsx

```diff
@@ -36,33 +36,80 @@
  * SOFTWARE.
  */
 import * as React from 'react';
+import { ServiceContext } from '@app/Shared/Services/Services';
+import { useSubscriptions } from '@app/utils/useSubscriptions';
 import { TargetView } from '@app/TargetView/TargetView';
-import { Card, CardBody, Tab, Tabs } from '@patternfly/react-core';
+import { Card, CardBody, Stack, StackItem, Tab, Tabs } from '@patternfly/react-core';
 import { EventTemplates } from './EventTemplates';
+import { AgentProbeTemplates } from '@app/Agent/AgentProbeTemplates';
+import { AgentLiveProbes } from '@app/Agent/AgentLiveProbes';
 import { EventTypes } from './EventTypes';
+import { concatMap, filter } from 'rxjs/operators';
+import { NO_TARGET } from '@app/Shared/Services/Target.service';
 
 export const Events = () => {
+  const context = React.useContext(ServiceContext);
+  const addSubscription = useSubscriptions();
   const [activeTab, setActiveTab] = React.useState(0);
+  const [enabled, setEnabled] = React.useState(false);
 
-  const handleTabSelect = (evt, idx) => {
-    setActiveTab(idx);
-  };
+  React.useEffect(() => {
+    addSubscription(
+      context.target
+        .target()
+        .pipe(
+          filter((t) => t !== NO_TARGET),
+          concatMap((_) => context.api.getActiveProbes(true))
+        )
+        .subscribe({
+          next: (_) => setEnabled(true),
+          error: (_) => setEnabled(false),
+        })
+    );
+  }, [addSubscription, context.api, context.target, setEnabled]);
+
+  const handleTabSelect = React.useCallback(
+    (evt, idx) => {
+      setActiveTab(idx);
+    },
+    [setActiveTab]
+  );
 
   return (
     <>
       <TargetView pageTitle="Events">
-        <Card>
-          <CardBody>
-            <Tabs activeKey={activeTab} onSelect={handleTabSelect}>
-              <Tab eventKey={0} title="Event Templates">
-                <EventTemplates />
-              </Tab>
-              <Tab eventKey={1} title="Event Types">
-                <EventTypes />
-              </Tab>
-            </Tabs>
-          </CardBody>
-        </Card>
+        <Stack hasGutter>
+          <StackItem>
+            <Card>
+              <CardBody>
+                <Tabs activeKey={activeTab} onSelect={handleTabSelect}>
+                  <Tab eventKey={0} title="Event Templates">
+                    <EventTemplates />
+                  </Tab>
+                  <Tab eventKey={1} title="Event Types">
+                    <EventTypes />
+                  </Tab>
+                </Tabs>
+              </CardBody>
+            </Card>
+          </StackItem>
+          {enabled ? (
+            <StackItem>
+              <Card>
+                <CardBody>
+                  <Tabs activeKey={activeTab} onSelect={handleTabSelect}>
```

Comment:
We might need to separate active-tab states here. Right now, a single activeTab states control 2 cards.

Reply:
This will unexpectedly close the open tab in other card.